lms/djangoapps/verify_student/views.py

```diff
@@ -1066,21 +1295,22 @@ def results_callback(request):
         return HttpResponseBadRequest(
             "Result {} not understood. Known results: PASS, FAIL, SYSTEM FAIL".format(result)
         )
-    incourse_reverify_enabled = InCourseReverificationConfiguration.current().enabled
-    if incourse_reverify_enabled:
-        checkpoints = VerificationCheckpoint.objects.filter(photo_verification=attempt).all()
-        VerificationStatus.add_status_from_checkpoints(checkpoints=checkpoints, user=attempt.user, status=status)
-        # If this is re-verification then send the update email
-        if checkpoints:
-            user_id = attempt.user.id
-            course_key = checkpoints[0].course_id
-            related_assessment_location = checkpoints[0].checkpoint_location
-
-            subject, message = _compose_message_reverification_email(
-                course_key, user_id, related_assessment_location, status, request
-            )
 
-            _send_email(user_id, subject, message)
+    checkpoints = VerificationCheckpoint.objects.filter(photo_verification=attempt).all()
+    VerificationStatus.add_status_from_checkpoints(checkpoints=checkpoints, user=attempt.user, status=status)
+
+    # If this is re-verification then send the update email
+    if checkpoints:
+        user_id = attempt.user.id
+        course_key = checkpoints[0].course_id
```

Comment:
Verification checkpoints are 1:m with Verification attempts, but this constraint is enforced in application code rather than in the database schema.  IIRC we used a m2m relation to decouple the VerificationCheckpoint table from the SoftwareSecurePhotoVerification table (avoiding adding a foreign key column to SoftwareSecurePhotoVerification).  Looking back, I'm not sure that was the right decision, but it's where we're at.